### ISSUE TITLE
[Suggestion] Option to Lock UV Mapping Scale?

### ISSUE THREAD

**Issue by Nianor** [JannisX11/blockbench#128]
Any chance we could get a "lock UV mapping scale" option? Something that would force all the UV mapped components to use a section of texture relative to the size of their actual faces? And you would rescale things globally instead of part by part? It would make it a lot harder to wind up with accidental texture scaling issues. I know that personally, one of the biggest issues I see with models from BlockBench is that a lot of them have moderate to severe textures stretching and distortion.
Basically, there would be a variable S, and if a cube is 1 by 2 by three, each face in the UV mapping is 1*S x 1*s, 1*S x 2*S, so on and so forth. This would be a fixed ratio for all faces, with config options to turn it on or off and change S.

**Comment by JannisX11** [comment]
We already have a locked auto uv option. I will see if introducing a setting for "S" works. Currently it is just 1:1.

**Comment by Nianor** [comment]
Cool, thanks! Must've missed that, but an option to control the scale would be greatly appreciated.

**Comment by Nianor** [comment]
Actually, looking at how the current auto UV option works, could there also be "lock UV scale" button that prevents resizing the texture mapping? I work with detailed enough models that when trying to move the UV mapping around by mouse, I frequently don't have the option to move, only resize.

**Comment by JannisX11** [comment]
Added in 3.0.5.
